notebooks/MAST/dwarf_galaxy_search/dwarf_galaxy_search.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<p> </p><blockquote>the tempurature</blockquote><p>the temperature</p><p> </p><p> </p><blockquote>the magnitudes in</blockquote><p>the magnitude in</p><p> </p><ul><li>there are multiple incidences of this case. There are times to write it in plural. But these incidences should be its singular form because it is a single filter's magnitude. Use the search function to find all such incidences in the notebook.</li></ul><p> </p><blockquote>Additionally, we'll take a look at the CMD in r-i. </blockquote><p>Please state why we need to see another CMD?</p><p> </p><p> </p><blockquote>y-axis</blockquote><p>vertical axis</p><blockquote>x-axis</blockquote><p>horizontal axis</p><p> </p><blockquote>the magnitudes of <strong>Leo I</strong> in g vs the magnitude of <strong>Leo I</strong> in g-r.</blockquote><p>the magnitude of <strong>Leo I</strong> in g vs the color of <strong>Leo I</strong> in g-r.</p><p> </p><blockquote>for g (visual filter or bluer filter) minus r (red filter)</blockquote><p>of g (bluer filter) minus r (red filter)</p><p> </p><blockquote>a couple arrays:</blockquote><p>a few numpy arrarys to store these magnitudes (g,r,i)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<blockquote>(red filter)</blockquote><p> </p><p>redder filter</p>